src/wp-admin/themes.php

```diff
@@ -555,7 +555,7 @@
 			?>
 			<a class="button activate" href="<?php echo $theme['actions']['activate']; ?>" aria-label="<?php echo esc_attr( $aria_label ); ?>"><?php _e( 'Activate' ); ?></a>
 			<?php
-			if ( ! $theme['blockTheme'] && current_user_can( 'edit_theme_options' ) && current_user_can( 'customize' ) ) {
+			if ( current_user_can( 'edit_theme_options' ) ) {
```

Comment:
[customize](https://wordpress.org/documentation/article/roles-and-capabilities/#customize) is included inside [edit_theme_options](https://wordpress.org/documentation/article/roles-and-capabilities/#edit_theme_options) as a permission, and customize does not apply to block themes since the customizer is not available to them. I don't think we need both in this case

Reply:
I agree with the earlier comment that the `customize` check ought to be retained for classic themes at least.

While the default approach in core is to map `customize` to `edit_theme_options`, the `map_meta_cap` filter can be used to alter this or disable the customizer all together.